rules/properties-order/README.md

```diff
@@ -32,9 +33,14 @@ Within an order array, you can include
 
 * unprefixed property names
 * group objects with these properties:
-	* `order: "flexible"`: If property isn't set (the default), the properties in this group must come in the order specified. If `"flexible"`, the properties can be in any order as long as they are grouped correctly.
+	* `order: "flexible"`: If property isn't set (the default), the properties in this group must come in the order 
+	specified. If `"flexible"`, the properties can be in any order as long as they are grouped correctly.
```

Comment:
Could you concatenate this list item, and the next one (about `emptyLineBefore`), to a single line, please?

Reply:
Yeah I haven’t done figured out what to write yet - after the weekend I’ll attempt to explain the feature to a friend and their feedback will help me work out what I need to include to sufficiently articulate what it is and how it works. 

I can’t just write it because I’m tainted with already knowing how it works.

Reply:
> I’ll attempt to explain the feature to a friend

That's a good plan! :)